filewriter_v2/README.md

```diff
@@ -0,0 +1,388 @@
+| Name | NeXus Type | Description |
+|---|---|---|
+| @NX_class | NX_CHAR | [`"NXentry"`](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-group)
+| [@default](https://manual.nexusformat.org/classes/base_classes/NXentry.html#nxentry-default-attribute) | NX_CHAR | Declares which NXdata group contains the data to be shown by default. |
+| [@version](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-version-attribute) | NX_CHAR | Describes the version of the NXmx definition used to write this data. |
+| [definition](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-definition-field) | NX_CHAR | NeXus NXDL schema to which this file conforms, `"NXmx"`. |
+| [end_time_estimated](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-end-time-estimated-field) | NX_DATE_TIME | Estimated time/date of the last data point collected in UTC, following ISO 8601 with suffix Z. Note that the time zone of the beamline is provided in `/entry/instrument/time_zone`. |
+| [program_name](https://manual.nexusformat.org/classes/base_classes/NXentry.html#nxentry-program-name-field) | NX_CHAR | Name of the program used to generate this file, `"DECTRIS-DAQ"`.
+| [program_name@version](https://manual.nexusformat.org/classes/base_classes/NXentry.html#nxentry-program-name-version-attribute) | NX_CHAR | Version number of the program used to generate this file. |
+| [start_time](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-start-time-field) | NX_DATE_TIME | ISO 8601 time/date of the first data point collected in UTC. |
+
+## /entry/data
+
+Describes the plottable data and related dimension scales. Please also refer to [Mutli-Channel Data](#multi-channel-data) for further details.
+
+| Name | NeXus Type | Description |
+|---|---|---|
+| @NX_class | NX_CHAR | [`"NXdata"`](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-data-group) |
+| [@axes](https://manual.nexusformat.org/classes/base_classes/NXdata.html#nxdata-axes-attribute) | NX_CHAR[4] | String array that defines the independent data fields used in the default plot for all of the dimensions of the `/entry/data/data` field, `"image_number, channel, ".", "."`. |
+| [@image_number_indices](https://manual.nexusformat.org/classes/base_classes/NXdata.html#nxdata-axisname-indices-attribute) | NX_INT | Indicates the dependency relationship of the `image_number` field with a dimensoin of the data, in this case `0`.
+| [@start_time_indices](https://manual.nexusformat.org/classes/base_classes/NXdata.html#nxdata-axisname-indices-attribute) | NX_INT | Indicates the dependency relationship of the `start_time` field with a dimensoin of the data, in this case `0`.
+| [@channel_indices](https://manual.nexusformat.org/classes/base_classes/NXdata.html#nxdata-axisname-indices-attribute) | NX_INT | Indicates the dependency relationship of the `channel` field with a dimensoin of the data, in this case `1`.
```

Comment:
also here "dimension" instead of dimensoin

Reply:
fixed in [34890fd](https://github.com/soph-dec/documentation/pull/1/commits/34890fdc03430a5b054a735918d0593e38a8e157)